bot/cogs/trivia.py

```diff
@@ -0,0 +1,449 @@
+import discord
+from discord.ext import commands
+import os
+import json
+import urllib.request
+import random
+
+correct = 'wrong'
+difficulty = 'easy'
+category = 'null'
+answers = []
+difficult = 'null'
+tokenID= '1234'
```

Comment:
any type of token should be loaded as a constant in settings.py, if this should be a secret, it should be an environment var.

Reply:
I have the code generate a token on the first run as part of getting data command, should I just leave that as a blank strng instead then to avoid confusion?